src/pydocstyle/checker.py

```diff
@@ -155,6 +155,7 @@ def check_docstring_missing(self, definition, docstring):
 
         """
         if (not docstring and definition.is_public or
+                docstring and docstring.lower().startswith('f') or
```

Comment:
Can we parenthesize these properly so that the conditions are obvious?

Reply:
Sure!